Core/HLE/proAdhoc.cpp

```diff
@@ -76,7 +76,7 @@ bool isLocalServer = false;
 uint8_t PPSSPP_ID = 0;
 sockaddr LocalhostIP;
 sockaddr LocalIP;
-int defaultWlanChannel = 1;
+int defaultWlanChannel = PSP_SYSTEMPARAM_ADHOC_CHANNEL_1; // Don't put 0(Auto) here, it needed to be a valid/actual channel number
```

Comment:
better use 11 on this one to match psp behavior #10698

Reply:
i was using 11 when i find out JPCSP using 11 as default, but later changed it back to 1 just to be different LOL